js/ui/context_menu/ui.menu_base.js

```diff
@@ -566,7 +566,7 @@ var MenuBase = HierarchicalCollectionWidget.inherit({
     },
 
     _hasSubmenu: function(node) {
-        return node.internalFields.childrenKeys.length;
+        return node.internalFields && node.internalFields.childrenKeys.length;
```

Comment:
If a node has no internalFields - it is an item. Why not converted items appear in this method? This issue should be fixed on another level